sdk/tdf_test.go

```diff
@@ -272,14 +310,37 @@ func (s *TDFSuite) Test_SimpleTDF() {
 	expectedTdfSize := int64(2058)
 	tdfFilename := "secure-text.tdf"
 	plainText := "Virtru"
-	{
-		kasURLs := []KASInfo{
-			{
-				URL:       "https://a.kas/",
-				PublicKey: "",
+
+	// add opts ...TDFOption to  TestConfig
+	testConfigs := []TestConfig{
+		{
+			tdfOptions: []TDFOption{
+				WithKasInformation(KASInfo{
+					URL:       "https://a.kas/",
+					PublicKey: "",
+				}),
+				WithMetaData(string(metaData)),
+				WithDataAttributes(attributes...),
 			},
-		}
+			tdfReadOptions: []TDFReaderOption{},
+		},
+		{
+			tdfOptions: []TDFOption{
+				WithKasInformation(KASInfo{
+					URL:       "https://d.kas/",
+					PublicKey: "",
+				}),
+				WithMetaData(string(metaData)),
+				WithDataAttributes(attributes...),
+				WithWrappingKeyAlg(ocrypto.EC256Key),
+			},
+			tdfReadOptions: []TDFReaderOption{
+				WithSessionKeyType(ocrypto.EC256Key),
+			},
+		},
+	}
 
+	for _, config := range testConfigs {
 		inBuf := bytes.NewBufferString(plainText)
```

Comment:
Better style here is to name these and make them explicit subtests, e.g. using `s.Run()`


https://pkg.go.dev/github.com/stretchr/testify@v1.10.0/suite#Suite.Run